context/app/static/js/components/Detail/ProvTabs.jsx

```diff
@@ -43,20 +48,20 @@ function ProvTabs(props) {
       <SectionHeader variant="h3" component="h2">Provenance</SectionHeader>
       <Paper className={classes.tabsRoot}>
         <Tabs
-          orientation="vertical"
-          variant="scrollable"
+          variant="standard"
           value={open}
           onChange={handleChange}
           aria-label="Detail View Tabs"
           className={classes.tabs}
-          indicatorColor="secondary"
+          tabColor="inherit"
+          TabIndicatorProps={{ style: { backgroundColor: '#9CB965' } }}
```

Comment:
Perhaps have this color defined at a higher level.

Reply:
Seems to be a one time use color for now, but I'll make an issue for it.